src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateOrgWithContactInformationDxAddress.java

```diff
@@ -397,7 +397,8 @@ void persists_organisation_with_white_spaces_removed() {
                         .lastName(" some-lname ")
                         .email(" someone@somewhere.com ")
                         .build())
-                .contactInformation(Arrays.asList(aContactInformationCreationRequest().addressLine1(" addressLine1 ")
+                .contactInformation(Arrays.asList(aContactInformationCreationRequest().uprn("uprn")
```

Comment:
please try to add multiple addresses and check organisation is getting persisted with multiple addresses along with uprn